tests/test_utils.py

```diff
@@ -0,0 +1,41 @@
+import os
+from autowebcompat import utils
+
+
+def test_mkdir():
+    direc_path = os.path.dirname(os.path.realpath(__file__)) + "test"
+    utils.mkdir(direc_path)
+    assert(os.path.exists(direc_path))
+
+
+label = {}
+
+
+def test_read_labels():
+    labels = utils.read_labels(file_name='labels.csv')
+    assert('labels' in locals() or 'labels' in globals())
```

Comment:
This assertion is strange. Just assert that labels is not null and that it is a dictionary.

Reply:
i was just asserting the existence of the variable

Reply:
You have defined it at line 15, so it clearly exists 😄